src/Uno.UI.Wasm/ts/Windows/Devices/Midi/Internal/WasmMidiAccess.ts

```diff
@@ -0,0 +1,37 @@
+﻿namespace Uno.Devices.Midi.Internal {
+	export class WasmMidiAccess {
+
+
+		private static midiAccess: WebMidi.MIDIAccess;
+
+		private static dispatchRequest: (hasAccess: boolean) => number;
+
+		private static initialize() {			
+			if (!this.dispatchRequest) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)